babel.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
This seems to be a config file similar to webpack.config.js.

You told me babel translates between different languages. So we need this because we use both vanilla js and vue js?

Babel presets seem to be "plugins" you call in

I have no idea what usebuiltins and corejs mean

But this seems like the wrong rabbit hole so I'll leave it

Reply:
> You told me babel translates between different languages. So we need this because we use both vanilla js and vue js?

I think in this case babel isn't translating the vue (but it probably could.)

It's more that there are different versions of javascript, eg Ecmascript 5 (ES5), ES6 ES7 etc and babel means you can write your code in a modern version of javascript and babel will convert it back to an older version that is supported in the browsers you're targeting.